dags/general/monitoring.py

```diff
@@ -0,0 +1,133 @@
+from airflow import DAG
+import json
+from operators.mattermost_operator import MattermostOperator
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.http_operator import SimpleHttpOperator
+from datetime import datetime
+from airflow_utils import send_alert
+from airflow.models.xcom import XCom
+from airflow.utils.db import provide_session
+from airflow.models import Variable
+import logging
+task_logger = logging.getLogger('airflow.task')
+
+
+"""
+This method returns list of extractions that are failing
+tap_exit_status = 1, failing
+tap_exit_status = 0, success
+tap_exit_status = None, extraction in process
+"""
+def stitch_check_extractions(response):
+    task_logger.info('Got extractions response from stitch api, checking for errors')
+    failed_extractions={}
+    try:
+        extractions = json.loads(response)['data']
+        for extraction in extractions:
+            if extraction['tap_exit_status'] is 1:
+                failed_extractions[extraction['source_id']] = extraction['tap_description']
+    except Exception as e:
```

Comment:
It's not advised to catch `Exception` (or even have bare `except`). More details are available on this [post](https://faun.pub/python-exception-handling-anti-patterns-6e42aacff251)  and on [this article](https://realpython.com/the-most-diabolical-python-antipattern/), as well as suggestions on alternatives.